### DISCUSSION TITLE
Profile selection on startup

### DISCUSSION THREAD

**Discussion by IromTypo** [runelite/runelite#16435] [Ideas]
Could an option be added to configure RuneLite to a profile selection screen on startup? There are settings that apply on restart, like custom windows chrome that I know of. It would be useful to players that use multiple accounts.

Kind regards,

Typo

**Comment by Alexsuperfly** [comment]
are you aware of the `--profile=<profile_name>` launch parameter, or do you want something else other than that?

**Reply by IromTypo** [reply]
I use the Jagex launcher, so that doesn't solve the problem. Unless there's a way to set profile_name as a variable? Thanks for replying.

**Comment by Alexsuperfly** [comment]
The newest launcher (version 2.6.3) creates a `RuneLite (Configure)` shortcut that when launched allows parameters to be configured in a gui interface and also enables the Jagex launcher to use them if it launches RuneLite.

**Reply by IromTypo** [reply]
I upgraded to 2.6.3, but I can't find it in Runelite (configure), what setting do you mean exactly?

**Reply by geheur** [reply]
For "client arguments" put `--profile=theprofilename`
n.b. `--profile theprofilename` does not work and the client will crash on launch.

it also doesn't fix this for the launcher since you'd have to modify it before every client launch.

**Reply by IromTypo** [reply]
Windows recommended me RuneLite (configure) for some reason, so now I get it, it's not in RuneLite in the settings of the main client. Yeah, precisely @geheur. Deeper integration of profiles with the launcher would be really nice.

**Comment by withoutgettingwet** [comment]
I'd like to make a suggestion for a stopgap measure if this is not currently possible: would it be possible to add a plugin with a whitelist that alerts you if the RSN account you're logging into doesn't match? That way, each profile could be set up to alert you if you're using the wrong one.

As nice as it would be to have it automatically switch, I'd be perfectly happy with just being reminded to switch manually.

**Reply by geheur** [reply]
are you aware that there is an auto switcher that auto switches on login? https://runelite.net/blog/show/2023-11-05-1.10.15-Release

**Reply by withoutgettingwet** [reply]
I was not, and this is perfect. Thank you!

**Reply by MoralFiber** [reply]
I don't see the auto switcher.  Instead, I see the cloud sync icon where the profile link icon should be.

**Reply by YvesW** [reply]
> I don't see the auto switcher.  Instead, I see the cloud sync icon where the profile link icon should be.  

Make sure you are logged in to the game.

**Reply by MoralFiber** [reply]
Honestly, thank you so much.  I spent so much time trying to figure out why it was missing while logged out on my 3 accounts... Thank you..
